### ISSUE TITLE
OSD task sequence fails on dynamically apply drivers step. "Error. Received 0x80072ee7 from WinHttpSendRequest"

### ISSUE THREAD

**Issue by RafYYC** [MSEndpointMgr/ModernDriverManagement#251]
Using mostly Dell devices (99.9%), problem started couple weeks ago. MECM version 2303 5.00.9106.1000.
We are able to image some desktop PCs with no issues, seems like laptops are the ones failing. That took me to think it is boot image drivers.
I have added drivers to the boot image (Dell WinPE 11 driver pack), removed old ones too, used a different network adapter to PXE boot (usually use Dell Docks, but I have tried using other USB gigabit adapters), still no luck. Also tried updating boot image from ADK option, made sure I applied updates to the Operating Systems Images (schedule updates option in MECM for both Win10 and Win11). 

Tried disabling "Modern Driver Management" step in the task sequence, the task sequence finishes without problems.

[log.txt](https://github.com/MSEndpointMgr/ModernDriverManagement/files/12188741/log.txt)

**Comment by alayac** [comment]
I didn't look at your log file very closely, but your issue description reminds me of a blog I read about how driver installation order matters in your WinPE boot image.  

Hopefully this helps.
https://www.deploymentresearch.com/back-to-basics-driver-installation-order-in-winpe-matters/

**Comment by RafYYC** [comment]
I finally figured it out, nothing wrong with Modern Drive Management, it was the "Realtek USB GbE Controller Family Driver". I initially suspected driver, so I went to Dell's site, downloaded the latest WinPE driver pack, and still had the same issue, turns out Dell is not putting in the latest driver from Realtek. After looking in other directions for a bit it occurred to me to go directly to Realtek, download the latest Win11 Driver (it is an installed but unpacked it with 7zip and used the folder Install_USB_Win11_11.13.0420.2023_20_05152023_05182023\WIN11\cx\64), added these drivers to MECM and to the Boot image, so far I have successfully finished imaging two laptops with 0 issues. The latest driver is version 1153.13.20.420 (the Dell driver pack had 1153.9.823.2022). Hope this helps others with Dell docks (WD19 and WD15). Thx
